Cargo.toml

```diff
@@ -9,24 +9,25 @@ license = "Apache-2.0"
 edition = "2018"
 
 [dependencies]
```

Comment:
I couldn't resist sorting the dependencies alphabetically.

Reply:
The only changes here are:
- changed `notify = "4.0.6"` to `notify = "5.0.0-pre.1"
- added `crossbeam-channel = "0.3.8"`